api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -40,8 +49,19 @@
 
     /**
      * Specify the fallback class to be used. An new instance of the fallback class
-     * is returned. The instance is unmanaged.
+     * is returned. The instance is unmanaged. The type parameter of the fallback class must be assignable to the
+     * return type of the annotated method. Otherwise, the {@link javax.enterprise.inject.spi.DeploymentException} must be thrown.
+     * 
      * @return the fallback class
      */
     Class<? extends FallbackHandler<?>> value();
```

Comment:
You have to introduce a dummy Noop class to define a default value for `value` in order to allow user to not specify value for `value`